tls/s2n_server_extensions.c

```diff
@@ -63,7 +64,9 @@ int s2n_server_extensions_send(struct s2n_connection *conn, struct s2n_stuffer *
         total_size += 5;
     }
 
-    total_size += s2n_kex_server_extension_size(conn->secure.cipher_suite->key_exchange_alg, conn);
+    if (conn->secure.cipher_suite->key_exchange_alg) {
```

Comment:
consider making a macro, such as we did for `s2n_server_can_send_sct_list` and friends:

in tls/s2n_tls.h:

```c
#define s2n_server_can_send_kex(conn) \
    ((conn)->secure.cipher_suite->key_exchange_alg)
```

it will make the code more readable to then say:

```c
if (s2n_server_can_send_kex(conn)) {
...
}
```

Reply:
yeah, good suggestion ;) Thanks!